apps/api/src/schema/enrollment-changes.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from "@hono/zod-openapi";
+import { terms, websocSectionTypes, websocStatuses } from "@packages/db/schema";
+import { yearSchema } from "./lib";
+
+export const enrollmentChangesQuerySchema = z.object({
+  sections: z
+    .string({ required_error: "The 'sections' query parameter is required." })
+    .nonempty("The 'sections' query parameter cannot be empty."),
```

Comment:
nonempty is deprecated. See https://zod.dev/?id=strings

```suggestion
.min(1, { message: "The 'sections' query parameter cannot be empty." })
```